README.md

```diff
@@ -1,17 +1,54 @@
 # Python Community News
+![PCN Logo](.github/assets/PCN%20Logo%20V0.16.jpg)
 
-
-### Watch the stream:
-
+- Check out <https://pythoncommunitynews.com>
 - [Watch the Show on Twitch](https://twitch.tv/kjaymiller)
 - [What the Shown on YouTube](https://youtube.com/c/kjaymiller)
+- https://buttondown.email/python-community-news
+## Topics
 
+The Python
```

Comment:
Not sure if this is left over from something else or if it's an incomplete sentence that needs finishing.